modules/virt-example-vm-node-placement-pod-affinity.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-specifying-nodes-for-vms.adoc
+
+[id="virt-example-vm-node-placement-pod-affinity_{context}"]
+= Example: VM node placement with pod affinity and pod anti-affinity
+
+In this example,
```

Comment:
In this example, VMs can only be scheduled on nodes that have a running pod with a label of "example-vm-key=example-vm-value". Scheduling is not possible if no such pod is running.

Additionally, the scheduler should prefer avoiding nodes with any pod that has a label of "example-vm-key-1=example-vm-value-1". Both constraints will be met if possible. However, if all nodes have a pod with this second label, it will be ignored for the purposes of scheduling.

*NOTE*: For the topologyKey, I strongly suggest we stick to labels listed on this page: https://kubernetes.io/docs/reference/labels-annotations-taints/ . In fact I think we should specifically use: `topologyKey: kubernetes.io/hostname`

Reply:
@ksimon1 can you keep me honest on this note about topologyKeys?